afs-core/src/main/java/com/powsybl/afs/AppFileSystemTool.java

```diff
@@ -160,7 +160,7 @@ private void runUnarchive(CommandLine line, ToolRunningContext context) {
                 throw new  AfsException("File system '" + fileSystemName + "' not found");
             }
             Path dir = context.getFileSystem().getPath(line.getOptionValue(DIR));
-            fs.getRootFolder().unarchive(dir);
+            fs.getRootFolder().unarchive(dir,true);
```

Comment:
why change the default ? (should be false ?, so no args)

Reply:
add option for itools